contract.php

```diff
@@ -269,3 +269,52 @@ function contract_civicrm_post($op, $objectName, $id, &$objectRef){
     }
   }
 }
+
+/**
+ * Adjust output of membership custom data for contract file download
+ * @param $content
+ * @param $context
+ * @param $tplName
+ * @param $object
+ */
+function contract_civicrm_alterContent(  &$content, $context, $tplName, &$object ) {
```

Comment:
Ok I can try the jquery method.  It's just there are no unique selectors so I will have to count td elements to get the right field.